core/tellie_server.py

```diff
@@ -399,7 +399,7 @@ def trigger_averaged(self):
         if self._firing is True:
             self.logger.warn("Cannot set trigger_averaged, already in firing mode.")
             return
-        if self._channel <= 56: #up to box 7
+        if self._channel[0] <= 56: #up to box 7
```

Comment:
Good catch! Could you also update this (and the b/p problem) in core/serial_command.py.